### ISSUE TITLE
Border radius change

### ISSUE THREAD

**Issue by DaitiDay** [danm36/zen-browser-private-browsing-toolbar-highlighting#6]
With the new 1.0.1-a.1 update the border radius around private window has to be tweaked to not get clipped.
![image](https://github.com/user-attachments/assets/3a42df92-3eca-480c-be66-c28fdab3e05f)

**Comment by danm36** [comment]
It look like this might be a bug in Zen itself, as the web page viewport seems to just ignore the border radius of any containing element (Something that didn't happen until Zen version 1.0.1-a1 as best as I can recall). 

I'll keep investigating whether I can do something, but it might not be possible right now and it might be better to raise a bug ticket on the main [Zen browser issues](https://github.com/zen-browser/desktop/issues) area. In fact, there does seem to be an issue - https://github.com/zen-browser/desktop/issues/1508 - but in my case the proposed fix doesn't seem to work.

**Comment by DaitiDay** [comment]
Can you open an issue on the browser repo explaining the problem?

**Comment by JLBlk** [comment]
Hey, i had the same problem with my SuperGradient Theme and was able to fix it by adding the following css, maybe it's working for you too

```
:root:not([inDOMFullscreen='true']):not([chromehidden~='location']):not([chromehidden~='toolbar']) {
  & #tabbrowser-tabbox #tabbrowser-tabpanels .browserSidebarContainer {
    overflow: hidden;
  }
}
```

**Comment by danm36** [comment]
Closing, as this appears to have been fixed in Zen version 1.0.1-a3

**Comment by astro-ray** [comment]
This problem is back for me in the new version.
